lib/matplotlib/artist.py

```diff
@@ -958,6 +958,27 @@ def set_alpha(self, alpha):
         """
         if alpha is not None and not isinstance(alpha, Number):
             raise TypeError('alpha must be a float or None')
+        if alpha is not None and (alpha < 0 or alpha > 1):
+            raise ValueError(f'alpha ({alpha}) is outside 0-1 range')
```

Comment:
codecov says you haven't tested these value errors.  Which is maybe a reasonable thing to test?

Reply:
I've added some; let's see if it is enough.